be/src/service/internal_service.cpp

```diff
@@ -168,6 +167,7 @@ void PInternalServiceImpl<T>::tablet_writer_add_batch(google::protobuf::RpcContr
     // exhausted, so we put this to a local thread pool to process
     int64_t submit_task_time_ns = MonotonicNanos();
     _tablet_worker_pool.offer([cntl_base, request, response, done, submit_task_time_ns, this]() {
+        SCOPED_SWITCH_BTHREAD();
```

Comment:
Why add here?
I think the thread in `_tablet_worker_pool` is not bthread?

Reply:
you are right, fix.